code/modules/client/preferences/middleware/quirks.dm

```diff
@@ -33,11 +33,19 @@
 
 	for (var/quirk_name in quirks)
 		var/datum/quirk/quirk = quirks[quirk_name]
+		var/datum/quirk_static_data/static_data = GLOB.all_quirk_static_data[quirk]
+		var/list/datum/preference/customization_options = static_data?.get_customization_data() || list()
+		var/customizable = FALSE
+		if (static_data)
+			customizable = static_data?.is_customizable()
```

Comment:
Don't bother with this variable--`static_data?.is_customizable()` is all you need.